checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/client/checkout/AccessCheckoutCVVClient.kt

```diff
@@ -61,7 +72,11 @@ class AccessCheckoutCVVClient private constructor(
             debugLog(TAG, "Making request to discover endpoint")
             val accessCheckoutDiscoveryClient = AccessCheckoutDiscoveryClientFactory.getClient()
 
-            accessCheckoutDiscoveryClient.discover(baseUrl = baseUrl, discoverLinks = DiscoverLinks.sessions)
+            accessCheckoutDiscoveryClient.discover(
+                baseUrl = baseUrl,
+                discoverLinks = DiscoverLinks.sessions
+            )
```

Comment:
You may want to move the creation of the discovery client in a private method to avoid duplication

Reply:
This will be refactored in a later PR so will take this into consideration later. Thanks for the suggestion though.